### PR TITLE
fix unsupported tokens alert in cart

### PR DESCRIPTION
**Pull request by chibie**
##### Refers/Fixes

fixes https://github.com/gitcoinco/web/issues/9790#issuecomment-983960087

### CODE REVIEW

**Review by PixelantDesign — COMMENTED** [review]
could you post a video @chibie ?

File: app/assets/v2/js/cart.js [hunk 1]
```diff
@@ -550,6 +550,23 @@ Vue.component('grants-cart', {
         }
       });
     },
+    /* Set supported tokens for L2s */
+    loadSupportedTokens() {
+      // zkSync
+      // Mainnet list: https://api.zksync.io/api/v0.1/tokens
+      // Rinkeby list: https://rinkeby-api.zksync.io/api/v0.1/tokens
+      this.zkSyncSupportedTokens = this.network === 'rinkeby'
```
Comment (gdixon) [inline app/assets/v2/js/cart.js:558]: Could we switch the order of one these ternaries so that this and `polygonSupportedTokens` are consistently defined?

**Review by gdixon — APPROVED** [review]
Looks and works great, just one tiny nitpick! 🚀

**Comment by thelostone-mc** [comment]
@chibie you'd also have to uncomment the code https://github.com/gitcoinco/web/commit/a07675dc94aa6391bf0c7543bc77b845c88b5c8e

**Comment by thelostone-mc** [comment]
let me push that actually

File: app/assets/v2/js/cart.js (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/v2/js/cart.js:558]: It does look like these both are always defined

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**